base/dsfs/dataset.go

```diff
@@ -509,6 +634,37 @@ func generateCommitDescriptions(prev, ds *dataset.Dataset, force bool) (short, l
 	return shortTitle, longMessage, nil
 }
 
+// copied from base/body.go to avoid circular dependency
```

Comment:
innnnnnteresting. we should think about reversing the flow of this, moving `ReadAllEntries` into dsfs

Reply:
I agree, will do in a follow-up change.